### ISSUE TITLE
Update KB article about events with section on creating recurring events

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#6073]
On hold until we implement recurring events, currently blocked by FE. 

## User Story or Problem Statement

As an editor of an event, i have documentation on creating recurring events. 

Existing KB articles
https://prod.cms.va.gov/help/vamc/about-events-content-for-vamcs
https://prod.cms.va.gov/help/cms-basics/how-to-add-a-new-event
https://prod.cms.va.gov/help/va-medical-centers-vamcs/about-events-content-for-vamcs/how-to-edit-an-existing-event

## Acceptance Criteria
- [ ] KB articles have been updated
- Combined `How to add a new event` and `How to edit an existing event` into a single article. Broke it up for scannability.  Sitting in a hold draft waiting for launch
- Leaving about-events-content-for-vamcs as is for now
- Added a draft of a new announcement article we can use for the email announcement link: https://prod.cms.va.gov/help/cms-announcements/new-recurring-events
- Set up feedback survey
- Added survey link to announcement article

**Post-sprint follow ups**
- Add issue for sending the email and put actual points on it

## Implementation steps
<optional>

## Design principles

Which [CMS design principle](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/cms-product-design-principles.md) is at play?
- [ ] Purpose-driven: Creates an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] Efficient: Removes distractions and create clear, straightforward paths to get the job done.
- [ ] Approachable: Offers friendly guidance over authoritative instruction.
- [ ] Consistent: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [x] Empowering: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Core Application Team`
- [x] `Product Support Team`

**Comment by cmaeng** [comment]
pushed back due to delayed release
